### ISSUE TITLE
Non reproducible NullReferenceException 

### ISSUE THREAD

**Issue by yannisgu** [fluentCODE/ScrawlIssues#40]
Not sure what I did to get this, but I have this Excpetion 2 times in the Windows Event Log:

```
Application: Scrawl.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NullReferenceException
Stack:
   at System.Windows.Forms.Control.MarshaledInvoke(System.Windows.Forms.Control, System.Delegate, System.Object[], Boolean)
   at System.Windows.Forms.Control.Invoke(System.Delegate, System.Object[])
   at System.Windows.Forms.WindowsFormsSynchronizationContext.Send(System.Threading.SendOrPostCallback, System.Object)
   at Microsoft.WindowsAPICodePack.Shell.ShellObjectWatcher.OnWindowMessageReceived(Microsoft.WindowsAPICodePack.Shell.WindowMessageEventArgs)
   at Microsoft.WindowsAPICodePack.Shell.MessageListenerFilter+RegisteredListener.MessageReceived(System.Object, Microsoft.WindowsAPICodePack.Shell.WindowMessageEventArgs)
   at Microsoft.WindowsAPICodePack.Shell.EventHandlerExtensionMethods.SafeRaise[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.EventHandler`1<System.__Canon>, System.Object, System.__Canon)
   at Microsoft.WindowsAPICodePack.Shell.MessageListener.WndProc(IntPtr, UInt32, IntPtr, IntPtr)
   at Microsoft.WindowsAPICodePack.Shell.Interop.ShellObjectWatcherNativeMethods.GetMessage(Microsoft.WindowsAPICodePack.Shell.Interop.Message ByRef, IntPtr, UInt32, UInt32)
   at Microsoft.WindowsAPICodePack.Shell.MessageListener.ThreadMethod()
   at System.Threading.ThreadHelper.ThreadStart_Context(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by stirno** [comment]
I think this is related to our ShellObjectWatcher not playing nice on shutdown.. so its hanging out in the background doing stuff after you've closed. Will investigate further.

**Comment by yannisgu** [comment]
Again such a strange one:

```
Application: Scrawl.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ComponentModel.InvalidAsynchronousStateException
Stack:
   at System.Windows.Forms.WindowsFormsSynchronizationContext.Send(System.Threading.SendOrPostCallback, System.Object)
   at Microsoft.WindowsAPICodePack.Shell.ShellObjectWatcher.OnWindowMessageReceived(Microsoft.WindowsAPICodePack.Shell.WindowMessageEventArgs)
   at Microsoft.WindowsAPICodePack.Shell.MessageListenerFilter+RegisteredListener.MessageReceived(System.Object, Microsoft.WindowsAPICodePack.Shell.WindowMessageEventArgs)
   at Microsoft.WindowsAPICodePack.Shell.EventHandlerExtensionMethods.SafeRaise[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]](System.EventHandler`1<System.__Canon>, System.Object, System.__Canon)
   at Microsoft.WindowsAPICodePack.Shell.MessageListener.WndProc(IntPtr, UInt32, IntPtr, IntPtr)
   at Microsoft.WindowsAPICodePack.Shell.Interop.ShellObjectWatcherNativeMethods.GetMessage(Microsoft.WindowsAPICodePack.Shell.Interop.Message ByRef, IntPtr, UInt32, UInt32)
   at Microsoft.WindowsAPICodePack.Shell.MessageListener.ThreadMethod()
   at System.Threading.ThreadHelper.ThreadStart_Context(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by stirno** [comment]
This should be fixed in next build (#181). Going to maybe-too-aggressively close it because the code freaking out won't even be in use now ;p
